cmake/IgnUtils.cmake

```diff
@@ -1804,3 +1804,73 @@ macro(_ign_cmake_parse_arguments prefix options oneValueArgs multiValueArgs)
   endif()
 
 endmacro()
+
+#################################################
+# ign_generate_conf()
+# Generate conf/*.yaml files for projects
+macro(ign_generate_conf)
```

Comment:
Can you change the name of the macro to be a little more descriptive? Maybe just including `ign-tools`  in the name is sufficient, eg. `ign_tools_generate_conf` or `ign_generate_conf_for_ign_tools`. @j-rivero might have better suggestions.

Reply:
We can by cycling on this but I still have the feeling that adding functions that are intrinsic related to gz-tools should be done in that repository. Since @harshmahesheka [mentioned](https://github.com/gazebosim/gz-tools/issues/72#issuecomment-1313247976) that the instruction in the Gazebo meeting was to use this repo, I'm going to try to accommodate that the decision here:

 * Following #344, I'll move these set of function to a new file.
 * Consider not to include the new file automatically into the `find_package(gz-cmake)` call and make it somehow optional.
 * The migration from Ign to Gz needs to be done
 * As @azeey suggested let's find a good name for them, something like `gztools_generate_subcommands_files` and consider to merge both features in once since seems to be that both ruby and yaml files are highly linked.